bench_manager/bench_manager/doctype/site/site.py

```diff
@@ -301,11 +301,13 @@ def create_site(site_name, install_erpnext, mysql_password, admin_password, key)
 	)
 	doc.insert()
 	frappe.db.commit()
-
+@frappe.whitelist()
 def jop_site_creation(commands, doctype, key):
     from bench_manager.bench_manager.utils import run_command
     run_command(commands=commands,doctype="Bench Settings",key=key)
     site = frappe.get_doc("Site",key)
     if site.developer_flag == 1:
             site.update_app_list()
-    site.developer_flag = 0
\ No newline at end of file
+    site.developer_flag = 0
```

Comment:
why do we force this even when there is a choice for users to set a developer mode?